docs/guides/sdks/android-sdk/install-android-sdk.md

```diff
@@ -123,6 +123,63 @@ If you are using ProGuard and you encounter a warning regarding our `AFKeystoreW
 ```groovy
 -keep class com.appsflyer.** { *; }
 -keep class kotlin.jvm.internal.** { *; }
+```
+## Backup rules
+
+The SDK's AndroidManifest.xml includes rules to opt out of backing up the Shared Preferences data. This is done to avoid retaining the same counters and AppsFlyer ID during reinstallation, thereby preventing the accurate detection of new installs or re-installs.
+
+To merge the SDK backup rules with your app backup rules and to prevent conflicts, perform the following instructions for each use case. 
+
+### fullBackupContent=”true”
+
+If you add `android:fullBackupContent="true"` inside the tag in the `AndroidManifest.xml`, you might get the following error:
+
+```
+Manifest merger failed : Attribute application@fullBackupContent value=(true)
+```
+
+To fix this error, add `tools:replace="android:fullBackupContent"` in the `<application>` tag in the `AndroidManifest.xml` file.
+
+### allowBackup=”false”
+
+If you add `android:allowBackup="false"`inside the tag in the `AndroidManifest.xml`, you might get the following error:
+
+```
+Error:
+	Attribute application@allowBackup value=(false) from AndroidManifest.xml:
+	is also present at [com.appsflyer:af-android-sdk:6.14.0] AndroidManifest.xml: value=(true).
+	Suggestion: add 'tools:replace="android:allowBackup"' to <application> element at AndroidManifest.xml to override.
+
+```
+
+To fix this error, add `tools:replace="android:allowBackup”` in the `<application>` tag in the `AndroidManifest.xml` file.
+
+### fullBackupContent="@xml/my_rules"
+
+If you’re also targeting **Android 11** and lower, and you have your own backup rules specified (`android:fullBackupContent="@xml/my_rules"`), in addition to the instructions above, please merge your backup rules with the AppsFlyer rules manually by adding the following rule:
+
+```xml AndroidManfiest.xml
+<full-backup-content>
+    ...//your custom rules
+    <exclude domain="sharedpref" path="appsflyer-data"/>
+</full-backup-content>
+
+```
+
+### dataExtractionRules="@xml/my_rules"
```

Comment:
The title should be "Android 12 and above" and be higher than the "Android 11 and lower"